airbyte-integrations/connectors/source-mixpanel/source_mixpanel/spec.json

```diff
@@ -8,9 +8,9 @@
     "properties": {
       "api_secret": {
         "order": 0,
-        "title": "Project Token",
```

Comment:
So confusion was from customer's end when we write this as `Token`. If so this change should not impact enabled Mixpanel connections.

Reply:
Mixpanel supports auth via both project token and project api secret, airbyte connector uses project secret as auth mechanism. So when they use token it gives improper error messages, but works properly with project secret.